soc_026_rw0_global_gender_gap/soc_026_rw0_global_gender_gap.py

```diff
@@ -0,0 +1,199 @@
+    political_a.columns = ['political_empowerment_subindex_rank', 'country', 'political_empowerment_subindex_score']
+    political_b = years_list['dataframes'][i][1].iloc[:,9:12] #second half
+    political_b.columns = ['political_empowerment_subindex_rank', 'country', 'political_empowerment_subindex_score']
+    political_c = pd.concat([political_a, political_b]).reset_index(drop=True) #concat
+    political_c['year'] = years_list['year'][i] #create a column for year
+    political_c = political_c.dropna(subset = ['country']) #drop NA
+    data_clean_subindex.append(political_c)
+    
+#    for item in range(len(data_clean_subindex)):
+#        print(item)
+#        while len(data_clean_subindex) > 4:
+#            df_subindex_merged.append(data_clean_subindex.pop(:4))
+
+#Merging all the subindicies for each year. Need to come up with a better way to do this... Tried in the comments above    
+df_subindex_2020 = reduce(lambda left,right: pd.merge(left, right, on=['year', 'country'], how = 'outer'), data_clean_subindex[0:4])
+df_subindex_2021 = reduce(lambda left,right: pd.merge(left, right, on=['year', 'country'], how = 'outer'), data_clean_subindex[4:])
+df_subindex_final = pd.concat([df_subindex_2020, df_subindex_2021])
+
+# Merge main index and subindex
+df_processed = pd.merge(data_main_index, df_subindex_final, on = ['year', 'country'], how = 'outer')
+df_processed = df_processed.replace(',','.', regex=True)
+        
+# Merge new years with old
+frames_carto_upload = [df_carto, df_processed]
+df_carto_upload = pd.concat(frames_carto_upload).reset_index(drop=True)
+
+#save processed dataset to csv
+processed_data_file = os.path.join(data_dir, dataset_name+'_edit.csv')
+df_carto_upload.to_csv(processed_data_file, index=False)
+
+'''
+Upload processed data to Carto
+'''
+logger.info('Uploading processed data to Carto.')
+util_carto.upload_to_carto(processed_data_file, 'LINK')
+
+'''
+Upload original data and processed data to Amazon S3 storage
+'''
+# initialize AWS variables
+aws_bucket = 'wri-public-data'
+s3_prefix = 'resourcewatch/'
+
+logger.info('Uploading original data to S3.')
+# Upload raw data file to S3
+
+# Copy the raw data into a zipped file to upload to S3
+raw_data_dir = os.path.join(data_dir, dataset_name+'.zip')
+with ZipFile(raw_data_dir,'w') as zip:
+    zip.write(raw_data_file, os.path.basename(raw_data_file))
+#Upload raw data file to S3
+uploaded = util_cloud.aws_upload(raw_data_dir, aws_bucket, s3_prefix+os.path.basename(raw_data_dir))
+
+logger.info('Uploading processed data to S3.')
+# Copy the processed data into a zipped file to upload to S3
+processed_data_dir = os.path.join(data_dir, dataset_name+'_edit.zip')
+with ZipFile(processed_data_dir,'w') as zip:
+    zip.write(processed_data_file, os.path.basename(processed_data_file))
+# Upload processed data file to S3
+uploaded = util_cloud.aws_upload(processed_data_dir, aws_bucket, s3_prefix+os.path.basename(processed_data_dir))
```

Comment:
Did you run this section? I don't see the data in S3, so I'm guessing you haven't yet.